parlai/tasks/squad/agents.py

```diff
@@ -196,10 +196,8 @@ def setup_data(self, path):
                     question = qa['question']
                     answers = (a['text'] for a in qa['answers'])
                     context = paragraph['context']
-                    yield (
-                        '\n'.join([title, context, question]),
-                        answers
-                    ), True
+                    yield ('\n'.join([title, context, question]),
```

Comment:
lol you don't like it?

Reply:
lolll I don't think this was me. Not sure what the rationale was, so I'll change it back.